### PR TITLE
Update C# SDK Binary to 2.1.8

### PR DESCRIPTION
**Pull request by lgarner**
C# DLL update to 2.1.8.

### CODE REVIEW

**Review by franklin-huang-mobiledgex — APPROVED** [review]

**Review by brucearmstrong — APPROVED** [review]

File: Runtime/Scripts/NetInterfaceIntegration.cs [hunk 1]
```diff
@@ -93,39 +95,49 @@ public string GetIPAddress(string sourceNetInterfaceName, AddressFamily addressf
...
     }
 
-    public bool HasCellular()
+    private bool testSameDefaultRoute(string iName)
     {
-      NetworkInterface[] netInterfaces = GetInterfaces();
-      bool foundByIp = false;
-
-      foreach (NetworkInterface iface in netInterfaces)
+      using (Socket socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, 0))
```
Comment (ah1053) [inline Runtime/Scripts/NetInterfaceIntegration.cs:100]: @lgarner  I am trying to understand this part 0 -> is ProtocolType.IP and your SocketType is UDP 

Screen Shot 2020-09-16 at 10 48 56 AM

**Review by ah1053 — APPROVED** [review]
Its working on Android S20 & iPhone 11 Pro Max ,Thanks Garner for the quick fix

File: Runtime/Scripts/NetInterfaceIntegration.cs [hunk 2]
```diff
@@ -93,39 +95,49 @@ public string GetIPAddress(string sourceNetInterfaceName, AddressFamily addressf
       return ipAddress;
     }
 
-    public bool HasCellular()
+    private bool testSameDefaultRoute(string iName)
```
Comment (ah1053) [inline Runtime/Scripts/NetInterfaceIntegration.cs:98]: nit: function name -> first letter upper case

File: Runtime/Scripts/NetInterfaceIntegration.cs [hunk 3]
```diff
@@ -93,39 +95,49 @@ public string GetIPAddress(string sourceNetInterfaceName, AddressFamily addressf
...
     {
-      NetworkInterface[] netInterfaces = GetInterfaces();
-      bool foundByIp = false;
-
-      foreach (NetworkInterface iface in netInterfaces)
+      using (Socket socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, 0))
       {
-        foreach (string cellularName in networkInterfaceName.CELLULAR)
+        // Should not actually connect if UDP (connectionless)
```
Comment (ah1053) [inline Runtime/Scripts/NetInterfaceIntegration.cs:102]: It's connecting what does that mean ?
Screen Shot 2020-09-16 at 10 54 57 AM

Comment (lgarner) [reply Runtime/Scripts/NetInterfaceIntegration.cs:102]: Local endpoint, from resolved default socket network route to host.

**Pull request merged**